.lagoon.yml

```diff
@@ -56,5 +56,5 @@ environments:
         service: auto-idler
       - name: curator
         schedule: '01 0 * * * '
-        command: /usr/bin/curator --config /curator/curator.yml /curator/actions.yml
+        command: . /home/.bashrc && /usr/bin/curator --config /curator/curator.yml /curator/actions.yml
```

Comment:
let's use `/lagoon/cronjob.sh` as this will not only source bash, but also run all other entrypoints, etc. :)